index.md

```diff
@@ -12,4 +12,4 @@ This website aims at bringing closer the spinal cord MRI community, which includ
 It provides a forum, workshop announcements (including material), common MRI acquisition protocols and a list of software dedicated to spinal cord MRI analysis.
 
 **[Subscribe to the mailing list](https://goo.gl/forms/Q425YRKwZP5tsExF2)** if you
-wish to be informed about those events or if you would like to join our [Slack Channel](https://spinalcordmri.slack.com).
+wish to be informed about those events or if you would like to join our [Slack Channel](https://publicslack.com/slacks/spinalcordmri/invites/new).
```

Comment:
if people join themselves, we don't need to go through the gform anymore so the sentence has to change, e.g.:

...wish to be informed about those events. Also, feel free to join our [Slack Channel](https://publicslack.com/slacks/spinalcordmri/invites/new) to engage with the spinal cord imaging community.

Reply:
Addressed in https://github.com/spinalcordmri/spinalcordmri.github.io/pull/33/commits/b1daed78b7aa5c00c30e8f870a817850374bcd79